gradle.properties

```diff
@@ -3,7 +3,7 @@ RELEASE_SIGNING_ENABLED=true
 
 GROUP=com.testsigma
 POM_ARTIFACT_ID=testsigma-java-sdk
-VERSION_NAME=1.2.11_cloud
+VERSION_NAME=1.2.12_cloud
```

Comment:
@SunilGembali Can we have this as 1.2.12_cloud_beta until qa finishes one round of testing. 
If we find any issues and want to make sdk changes, we need to change the number or version again. It won't be good